utils/ipc.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, *args: Any, **kwargs: Any):
         # callbacks are subscribed to event name and message id
         self._callbacks: Dict[str, Dict[str, asyncio.Future[IPCData]]] = {}
         # event handlers are subscribed to event name
-        self._event_handlers: Dict[str, Set[_HandlerType]] = {}
+        self._event_handlers: Dict[str, List[_HandlerType]] = []
```

Comment:
`_event_handlers` is a Dict why is it a List 💀

Reply:
oh shit